src/app/components/App.js

```diff
@@ -4,6 +4,7 @@ import Loading from "./Loading";
 
 const LandingScreen = lazy(() => import(/* webpackPrefetch: true, webpackChunkName: "LandingScreen" */ "../screens/LandingScreen"));
 const NotFoundScreen = lazy(() => import(/* webpackPrefetch: true, webpackChunkName: "NotFoundScreen" */ "../screens/NotFoundScreen"));
+const ArticleScreen = lazy(() => import(/* webpackPrefetch: true, webpackChunkName: "ArticleScreen" */ '../screens/ArticleScreen'));
 
```

Comment:
Change the single quotes to double quotes for this import to match the other import styles.